indexing-hadoop/src/main/java/io/druid/indexer/hadoop/DatasourceIngestionSpec.java

```diff
@@ -108,19 +112,66 @@ public QueryGranularity getGranularity()
     return metrics;
   }
 
+  @JsonProperty
+  public boolean getIgnoreWhenNoSegments()
```

Comment:
should this be `isIgnoreWhenNoSegments` ? Not sure if it make a different with jackson or not, but just for consistency.